packages/components/src/components/CodeEditor/tooltips/ShowType.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC } from "react";
+
+import { Type } from "@quri/squiggle-lang";
+
+export const ShowType: FC<{ type: Type }> = ({ type }) => {
+  return (
+    <div className="max-w-80 font-mono text-xs text-slate-600">
+      {type.toString()}
```

Comment:
TODO: `type.toString()` is not indented and not colored, we should traverse it recursively.